migrator/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM migrate/migrate
```

Comment:
What is this container intended for?

Reply:
Docker image for golang-migrate binary https://github.com/golang-migrate/migrate

Reply:
You can specify it in docker-compose.yaml.